tests/queries/0_stateless/01316_create_user_syntax_hilite.reference

```diff
@@ -1 +0,0 @@
-[1mCREATE USER[0m user[1m IDENTIFIED[0m[1m WITH plaintext_password[0m[1m BY[0m 'hello'
```

Comment:
Let's restore this test. Later we can think about removing it but currently it's better to see the changes in hiliting this PR makes.

Reply:
I've moved this test here

https://github.com/murfel/ClickHouse/blob/2c1a311478b202724a84abd5a80a94f4b66709c8/src/Parsers/tests/gtest_format_hiliting.cpp#L127

where it will be tested with a hilite-aware comparator

Reply:
It's not exactly the same, so I think it's better to keep both tests for now.